### PR TITLE
Run stage1 and 2 of futurize on all python files

### PR DESCRIPTION
**Pull request by mri-dula**
As part of the [python upgrade ](https://atsico.atlassian.net/browse/RBE-783)epic, this PR is to modify the pytasty package to be compatible with python 3 and also back-compatible with python 2. 

### JIRA Card
https://atsico.atlassian.net/browse/RBE-1013

### Reference Links
https://python-future.org/automatic_conversion.html#futurize-py2-to-py2-3

### CODE REVIEW

**Review by nraval1729 — APPROVED** [review]
@mri-dula Can you also update the `version` to `0.2.0` in `setup.py`?

File: pytasty/request_handler.py [hunk 1]
```diff
@@ -46,7 +49,7 @@ def clean_url(url):
     cleaned_url = splits[0]
     if splits[1:]:
         params = {key: list(set(val))
-                  for key, val in urlparse.parse_qs(splits[1]).items()}
+                  for key, val in list(urllib.parse.parse_qs(splits[1]).items())}
```
Comment (nraval1729) [inline pytasty/request_handler.py:52]: @Khusbu Can you explain the urlparse issue that could happen for this line?

Comment (Khusbu) [reply pytasty/request_handler.py:52]: I faced issue in this code:
```
    request_line = urllib.parse.urlunparse(
        (None, None, parsed.path, parsed.params, parsed.query, None)
        )
```
It should be
```
    request_line = urllib.parse.urlunparse(
        ('', '', parsed.path, parsed.params, parsed.query, '')
        )
```

Comment (Khusbu) [reply pytasty/request_handler.py:52]: The snippet you shared should work fine.

Comment (nraval1729) [reply pytasty/request_handler.py:52]: Cool, thanks 👍

**Review by nraval1729 — APPROVED** [review]

**Pull request merged**